packages/create-astro/src/index.ts

```diff
@@ -68,6 +72,22 @@ export async function main() {
     process.exit(1);
   }
 
+  // Post-process in parallel
+  await Promise.all(
+    POSTPROCESS_FILES.map(async (file) => {
+      const fileLoc = path.join(cwd, file);
+
+      switch (file) {
+        case 'package.json': {
+          const packageJSON = JSON.parse(await fs.promises.readFile(fileLoc, 'utf8'));
+          delete packageJSON.snowpack; // delete snowpack config only needed in monorepo (can mess up projects)
```

Comment:
Unrelated fix: we have some artifacts in `package.json` from our monorepo since we decided to use the internal examples as public templates. This cleans up the `snowpack` setting which may break projects.